### PR TITLE
lobbymusc

### PR DESCRIPTION
**Pull request by quadbium**
## About the Pull Request

adds
[SCP Original] Astowo - The Bookburners V2 (Theme of the ̶G̶O̶C̶ MTT)
https://www.youtube.com/watch?v=NXZ4hbOl7eY
to the lobby playlist
## Why It's Good For The Game
it was too good/funny to leave out

## Changelog

:cl:
add: adds the bookburners v2 to lobby music
/:cl:

### CODE REVIEW

**Comment by ChuChu42** [comment]
**Fabulous**

File: sounds/music/The_Bookburners_V2.ogg
Comment (checkraisefold) [inline sounds/music/The_Bookburners_V2.ogg:1]: make sure you have ran optivorbis on this

**Review by checkraisefold — APPROVED** [review]
approved as long as you have done whats said in the review

File: sounds/music/The_Bookburners_V2.ogg
Comment (quadbium) [reply sounds/music/The_Bookburners_V2.ogg:1]: I did, still not automated then?

**Pull request merged**